### ISSUE TITLE
No option Connect to hidden network

### ISSUE THREAD

**Issue by Daniel20000522** [ubports/ubuntu-touch#1201]
- Device: Nexus 4
- Channel: edge
- Build: 2019-08-22

### Steps to reproduce
1. Install Ubuntu Touch with the Wipe Data option (so that you can go through the system configuration)
2. During configuration, the system will ask you if you want to connect to WiFi.

### Expected behavior
I can connect to a hidden network.

### Actual behavior
There is no option to connect to a hidden network.

**Comment by NeoTheThird** [comment]
Can someone test if this is a regression (i.e. can you connect to hidden wifi on stable)?

**Comment by UniversalSuperBox** [comment]
No, this does not appear to be a regression. The Wizard does not have an option to connect to a hidden network and apparently never has.

![image](https://user-images.githubusercontent.com/21966173/64081506-b0e70880-ccc7-11e9-8ab3-fa2d1a8583de.png)

**Comment by Daniel20000522** [comment]
Shouldn't the option to connect to a hidden network be added?

**Comment by Flohack74** [comment]
Yeah we can make a feature request out of it.
